db_populate.py

```diff
@@ -0,0 +1,224 @@
+# This file is used to populate the database(mainly for testing)
+from lemur.utility import *
```

Comment:
Then do `from lemur import utility as u`, and then do `u.<method name>` for each one.